<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Количественная оценка случайности</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Алгоритмический:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Табличный:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="alg_table">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>1 разр.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2 разр.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3 разр.</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>1 разр.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2 разр.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3 разр.</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Количественная оценка случайности:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_alg">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_table">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Введите последовательность чисел для проверки через пробел:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manual_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>41</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_manual">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Количественная оценка случайности:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Количественная оценка случайности:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fil_manualy">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Способы:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
